### ISSUE TITLE
Both of the Campuses Filters in DataViews do not Allow Editing

### ISSUE THREAD

**Issue by treyhendon** [SparkDevNetwork/Rock#2618]
### Prerequisites
* [x] Put an X between the brackets on this line if you have done all of the following:
    * Can you reproduce the problem on a fresh install or the [demo site](http://rock.rocksolidchurchdemo.com/)?
    * Did you include your Rock version number and [client culture](https://github.com/SparkDevNetwork/Rock/wiki/Environment-and-Diagnostics-Information) setting?
    * Did you [perform a cursory search](https://github.com/issues?q=is%3Aissue+user%3ASparkDevNetwork+-repo%3ASparkDevNetwork%2FSlack) to see if your bug or enhancement is already reported?

### Description

Editing or creating a new DataView using the Campuses or Campuses (Active Only) causes an exception of `Input string not in the correct format`.

### Steps to Reproduce

1. Create a new Person DataView
2. Add a Filter Type of Campuses

**Expected behavior:** 

A checkbox list of Campuses to select should be displayed.

**Actual behavior:** 

```
Error
Input string was not in a correct format.
   at System.Text.StringBuilder.FormatError()
   at System.Text.StringBuilder.AppendFormatHelper(IFormatProvider provider, String format, ParamsArray args)
   at System.String.FormatHelper(IFormatProvider provider, String format, ParamsArray args)
   at Rock.Reporting.DataFilter.Person.CampusesFilter.GetClientFormatSelection(Type entityType)
   at Rock.Web.UI.Controls.FilterField.RenderControl(HtmlTextWriter writer)
   at Rock.Web.UI.Controls.FilterGroup.RenderControl(HtmlTextWriter writer)
   at System.Web.UI.Control.RenderChildrenInternal(HtmlTextWriter writer, ICollection children)
   at System.Web.UI.Control.RenderControlInternal(HtmlTextWriter writer, ControlAdapter adapter)
   at System.Web.UI.Control.RenderChildrenInternal(HtmlTextWriter writer, ICollection children)
   at System.Web.UI.HtmlControls.HtmlContainerControl.Render(HtmlTextWriter writer)
   at System.Web.UI.Control.RenderControlInternal(HtmlTextWriter writer, ControlAdapter adapter)
   at System.Web.UI.Control.RenderChildrenInternal(HtmlTextWriter writer, ICollection children)
   at System.Web.UI.WebControls.WebControl.RenderContents(HtmlTextWriter writer)
   at System.Web.UI.WebControls.WebControl.Render(HtmlTextWriter writer)
   at System.Web.UI.Control.RenderControlInternal(HtmlTextWriter writer, ControlAdapter adapter)
   at System.Web.UI.Control.RenderChildrenInternal(HtmlTextWriter writer, ICollection children)
   at System.Web.UI.Control.RenderControlInternal(HtmlTextWriter writer, ControlAdapter adapter)
   at System.Web.UI.Control.RenderChildrenInternal(HtmlTextWriter writer, ICollection children)
   at System.Web.UI.UpdatePanel.RenderChildren(HtmlTextWriter writer)
   at System.Web.UI.UpdatePanel.Render(HtmlTextWriter writer)
   at System.Web.UI.Control.RenderControlInternal(HtmlTextWriter writer, ControlAdapter adapter)
   at System.Web.UI.PageRequestManager.RenderFormCallback(HtmlTextWriter writer, Control containerControl)
   at System.Web.UI.Control.RenderChildrenInternal(HtmlTextWriter writer, ICollection children)
   at System.Web.UI.HtmlControls.HtmlForm.RenderChildren(HtmlTextWriter writer)
   at System.Web.UI.HtmlControls.HtmlContainerControl.Render(HtmlTextWriter writer)
   at System.Web.UI.Control.RenderControlInternal(HtmlTextWriter writer, ControlAdapter adapter)
   at System.Web.UI.HtmlControls.HtmlForm.RenderControl(HtmlTextWriter writer)
   at System.Web.UI.PageRequestManager.RenderPageCallback(HtmlTextWriter writer, Control pageControl)
   at System.Web.UI.Control.RenderChildrenInternal(HtmlTextWriter writer, ICollection children)
   at System.Web.UI.Page.Render(HtmlTextWriter writer)
   at System.Web.UI.Control.RenderControlInternal(HtmlTextWriter writer, ControlAdapter adapter)
   at System.Web.UI.Page.ProcessRequestMain(Boolean includeStagesBeforeAsyncPoint, Boolean includeStagesAfterAsyncPoint)
```

### Versions

* **Rock Version:** Rock McKinley 6.10  
* **Client Culture Setting:** en-US

**Comment by cabal95** [comment]
Confirmed still exists on prealpha server too.

**Comment by treyhendon** [comment]
Just tested a 6.9 install and this is not an issue there.  I'm guessing something changed in the way the field type is parsing the saved attribute value?

**Comment by edmistj** [comment]
@treyhendon thanks for clarifying.

**Comment by MikeDPeterson** [comment]
fixed with https://github.com/SparkDevNetwork/Rock/commit/a0313912cdb67c62f4743245fc433655554ec57d

**Comment by mcnoodles** [comment]
Any chance this could be fixed and released in 6.11?  We are struggling without the filter in v 6.10 and aren't ready to upgrade to v 7.X yet.
